app/lib/publish_constraint.rb

```diff
@@ -2,6 +2,6 @@
 
 class PublishConstraint
   def matches?(request)
-    Settings.base_url&.include?(request.host) || Settings.publish_api_url&.include?(request.host) || request.host.include?('publish-teacher-training-pr')
+    Settings.base_url&.include?(request.host) || Settings.publish_api_url&.include?(request.host) || request.host.include?('publish-teacher-training-pr') || request.host.include?('publish-review')
```

Comment:
```suggestion
    Settings.base_url&.include?(request.host) || Settings.publish_api_url&.include?(request.host) ||  request.host.include?('publish-review')
```

If we removing `cf` then we can remove `publish-teacher-training-pr`?

Reply:
We can remove this when PaaS is decommissioned, but we need both while we dual run.